src/components/SceneApp/Queries/ClusterMappingQueries.ts

```diff
@@ -211,18 +211,16 @@ function getNodesReadyFieldConfig() {
         cellComponent: (props) => {
             const values = (props.value as string)?.split("/").map((v: string) => parseInt(v, 10));
             if (!!values) {
-                const iconName = values[0] === values[1] ? "check-circle" : "exclamation-circle"
-                const color = values[0] === values[1] ? "green" : "red"
-                return React.createElement(
-                    'div',
-                    {},
-                    React.createElement(Icon, { name: iconName, style: { color: color } }),
-                    ` ${(props.value as string)}`
+                const iconName = values[0] === values[1] ? "check-circle" : "exclamation-circle";
```

Comment:
Check if the values array length is 2 or else value[1] could throw an exception

Reply:
done :)